webpack/JobInvocationDetail/TemplateInvocationPage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import PageLayout from 'foremanReact/routes/common/PageLayout/PageLayout';
+import { translate as __, sprintf } from 'foremanReact/common/I18n';
+import { TemplateInvocation } from './TemplateInvocation';
+import { selectTemplateInvocation } from './JobInvocationSelectors';
+import { jobInvocationDetailsUrl } from './JobInvocationConstants';
+
+const TemplateInvocationPage = ({
+  match: {
+    params: { hostID, jobID },
+  },
+}) => {
+  const description = sprintf(__('Template Invocation for %s'), jobID);
```

Comment:
![image](https://github.com/user-attachments/assets/a0897b6f-6728-47b9-a884-ab1a9cddf6e6)

Does the heading even need to be there?

Reply:
the old page has it in the breadcrumbs 
![image](https://github.com/user-attachments/assets/a717bd3b-29b8-43a6-851f-4fdec6c73b92)

Reply:
yup, done